static/app/views/performance/landing/utils.tsx

```diff
@@ -37,31 +37,6 @@ export enum LandingDisplayField {
 }
 
 export const LANDING_DISPLAYS = [
-  {
-    label: 'All Transactions',
-    field: LandingDisplayField.ALL,
-  },
-  {
-    label: 'Frontend (Pageload)',
-    field: LandingDisplayField.FRONTEND_PAGELOAD,
-  },
-  {
-    label: 'Frontend (Other)',
-    field: LandingDisplayField.FRONTEND_OTHER,
-  },
-  {
-    label: 'Backend',
-    field: LandingDisplayField.BACKEND,
-  },
-  {
-    label: 'Mobile',
-    field: LandingDisplayField.MOBILE,
-    isShown: (organization: Organization) =>
-      organization.features.includes('performance-mobile-vitals'),
-  },
-];
-
-export const LANDING_V3_DISPLAYS = [
```

Comment:
renamed `LANDING_V3_DISPLAYS` -> `LANDING_DISPLAYS` and removed old `LANDING_DISPLAYS`